### ISSUE TITLE
SharePoint SE Installation Failed within W2022 Server Core

### ISSUE THREAD

**Issue by vherrlein** [brianlala/AutoSPInstaller#79]
Dear,

Because of the way how the server core is working, we cannot create shortcut.

1. May be because of SE setup.exe now another addition argument must be added "/IAcceptTheLicenseTerms".
Otherwise the script raise an exception saying the setup installer required a reboot.
https://github.com/brianlala/AutoSPInstaller/blob/b14be321bbd3f431b81608e0c0f658f55e89e3ac/SP/Automation/AutoSPInstallerModule.psm1#L996

2. The conditional statement at the following line should also check if the current machine is a server core:
https://github.com/brianlala/AutoSPInstaller/blob/b14be321bbd3f431b81608e0c0f658f55e89e3ac/SP/Automation/AutoSPInstallerMain.ps1#L205

Note: How to retrieve the server product info with the server level : [Determining Whether Server Core Is Running](https://docs.microsoft.com/en-us/previous-versions/windows/desktop/legacy/hh846315(v=vs.85))

**Comment by vherrlein** [comment]
In addition the config XML for server core must change as following:

1. Display Level must be **none** 
  `<Display Level="none" CompletionNotice="No" AcceptEula="Yes"/>`
https://github.com/brianlala/AutoSPInstaller/blob/b14be321bbd3f431b81608e0c0f658f55e89e3ac/SP/Automation/AutoSPInstallerModule.psm1#L316

2. USINGUIINSTALLMODE must be "0"
`<Setting Id="USINGUIINSTALLMODE" Value="0"/>`
https://github.com/brianlala/AutoSPInstaller/blob/b14be321bbd3f431b81608e0c0f658f55e89e3ac/SP/Automation/AutoSPInstallerModule.psm1#L321

**Comment by Coco0185** [comment]
Hi @vherrlein and @brianlala ,

I have to install sharepoint SE on Windows Server 2022.

@vherrlein : Does the script work with your last posted modification?

@brianlala  : Is it possible to use your script for the installation of SharePoint "SE" now?


Best regards

**Comment by brianlala** [comment]
Yes, SE and Server 2022 are both supported for AutoSPInstaller. 

Note though that https://autospinstaller.com has not been updated yet to support building XML configs for the most recent changes, but the scripts themselves will work on the latest version/OS.
